conf/ocsci/production_powervs_upi.yaml

```diff
@@ -0,0 +1,30 @@
+RUN:
+  bin_dir: '/usr/local/bin'
+
+DEPLOYMENT:
+  local_storage: True
+  force_download_installer: False
+  force_download_client: False
+  ocs_csv_channel: "stable-4.6"
```

Comment:
we should not put the version in the platform conf file. For picking the version when executing the test, you should append `--ocsci-conf conf/ocs_version/ocs-4.6.yaml`

Reply:
@ebenahar you should instead use `--ocs-version 4.6`, the [README](https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocs_version/README.md) explains why.

Reply:
right, thanks Coady

Reply:
Ok, i see you can invoke run-ci with multiple --ocsci-conf parameters... Sure, iIwill make that change.

Reply:
I will use --ocs-version 4.6.

Reply:
Done.